packages/demo-protobuf/src/msgs.ts

```diff
@@ -0,0 +1,27 @@
+import { Message } from "protobufjs";
+
+import { cosmosField, cosmosMessage } from "./decorator";
+import { Registry } from "./registry";
+
+export const defaultRegistry = new Registry();
+
+@cosmosMessage(defaultRegistry, "/cosmos.Coin")
+export class Coin extends Message<{}> {
+  @cosmosField.string(1)
+  public readonly denom?: string;
+
+  @cosmosField.string(2)
+  public readonly amount?: string;
+}
+
+@cosmosMessage(defaultRegistry, "/cosmos.bank.MsgSend")
```

Comment:
Just saw this. This is amazing!

You gotta write this up somewhere when it is finished.

Reply:
:D Let's see if a chain accepts it